_data/simple-icons.json

```diff
@@ -17451,6 +17451,11 @@
             "hex": "84BD00",
             "source": "https://www.yubico.com/wp-content/themes/coronado/img/icon.svg"
         },
+        {
+            "title": "YunoHost",
+            "hex": "000000",
+            "source": "https://icon.icepanel.io/Technology/svg/Yuno-Host.svg"
```

Comment:
Is IcePanel an official source for YunoHost? I can't see anything on the homepage to suggest it is.
Looking at Yunohost's own documentation there _is_ an SVG - but it's a horizontal one.

Reply:
Nah, IcePanel is not really an official source for YunoHost. I just thought I could save some work here as on their official website, they just have a png, see https://yunohost.org. As I found the svg on IcePanel which looked fine I didn't want to manually vectorize the png icon from their website.

So, what would be the correct procedure here? Manually vectorize the png from the website or just change the source? Personally, I prefer the compact logo over the horizontal logo from their documentation.

Reply:
Ideally, we'd find an official source (one of their websites, a Wikipedia entry, or a GitHub repo of theirs) and use that as a source. Failing that, we can compare the SVG you've found against the PNG - but the source should _really_ come from the company themselves.

Reply:
:eyes: https://github.com/YunoHost/doc/blob/master/images/ynh_logo_black.svg